packages/components/src/List/ListComposition.stories.js

```diff
@@ -246,7 +246,7 @@ storiesOf('Data/List/List Composition', module)
 >
 	<List.Toolbar>
 		<List.Toolbar.Right>
-			<List.TextFilter id="my-list-textFilter" />
+			<List.TextFilter id="my-list-textFilter" applyOn={['name', 'description']} />
```

Comment:
Could you add a specific story for that ? Or ar least a note in the story description ?